### PR TITLE
LPS-169375 Fix expiring policies chart

### PR DESCRIPTION
**Pull request by joanagrego**
Related with --> [LPS-169375](https://issues.liferay.com/browse/LPS-169375)

### CODE REVIEW

**Comment by joanagrego** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/policies-expiring/index.js [hunk 1]
```diff
@@ -171,9 +183,14 @@ export default function () {
 				).then((results) => {
 					const expiringPoliciesResultThreeMonths = results?.data;
 
-					setThreeMonthsTotalPolicies(
-						expiringPoliciesResultThreeMonths?.totalCount
+					const arrayThreeMonthsExpiringPolicies =
+						expiringPoliciesResultThreeMonths?.items;
+
+					setThreeMonthsExpiringPolicies(
+						expiringPoliciesResultThreeMonths.totalCount
```
Comment (eliasantosf) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/policies-expiring/index.js:190]:
```suggestion
						expiringPoliciesResultThreeMonths?.totalCount
```

**Review by silasbsi — APPROVED** [review]
LGTM :heavy_check_mark:

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/policies-expiring/index.js [hunk 2]
```diff
@@ -148,16 +123,53 @@ export default function () {
...
 						}
-
-						setThisMonthTotalPolicies(
-							totalFilteredPolicies?.totalCount
-						);
 					}
 				);
 
-				setChartTitle(arrayOfExpiringPolicies?.length?.toString());
+				setChartTitle(arrayOfexpiringPolicies?.length?.toString());
```
Comment (eliasantosf) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/policies-expiring/index.js:129]:
```suggestion
				setChartTitle(arrayOfexpiringPolicies?.length.toString());
```

**Review by eliasantosf — CHANGES_REQUESTED** [review]

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/policies-expiring/index.js (hunk 1 above)
Comment (joanagrego) [reply modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/policies-expiring/index.js:190]: Done!

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/policies-expiring/index.js (hunk 2 above)
Comment (joanagrego) [reply modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/policies-expiring/index.js:129]: Done!

**Review by eliasantosf — APPROVED** [review]
LGTM :fire: Thanks @joanagrego

**Comment by silasbsi** [comment]
ci:forward

**Comment by silasbsi** [comment]
ci:forward

**Pull request closed without merging**